unime/src/routes/(app)/scan/utils.ts

```diff
@@ -5,7 +5,7 @@ import { info, warn } from '@tauri-apps/plugin-log';
 export const checkScanPrerequisites = async (): Promise<boolean> => {
   return await checkPermissions()
     .then((permission: PermissionState) => {
-      info(`app has permissions to access the camera: ${res}`);
+      info(`App has permissions to access the camera: ${permission}`);
```

Comment:
Minor suggestion...
Since you capitalized all `info` log messages, for completeness you might want to do the same for `debug` and `warn`. 

You can find them easily by searching for `` debug(` `` and `` warn(` ``.

Reply:
I agree, I will capitalize the other log statements as well. There are some `debug` statements where it doesn't make sense, such as:
```
debug(`assetUrl: ${assetUrl}`);`
```